packages/salesforcedx-vscode-core/src/conflict/persistentStorageService.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { FileProperties } from '@salesforce/source-deploy-retrieve/lib/src/client/types';
```

Comment:
Can we import `FileProperties` from the top level? ie.

`import { FileProperties } from '@salesforce/source-deploy-retrieve';`

Reply:
Yes, with yesterday's library release you should be able to do this.

Reply:
Would need the extensions to import SDR 2.1.5 which is currently causing me problems, perhaps related to @salesforce/core. Will update this as soon as SDR 2.1.5 is being used